pkg/db/db.go

```diff
@@ -87,7 +87,7 @@ func Init(dbDir string, opts ...Option) (err error) {
 			if err = os.Remove(dbPath); err != nil {
 				return
 			}
-			db, err = bolt.Open(dbPath, 0644, dbOptions.boltOptions)
+			err = eb.Errorf("failed to open db: %s", r)
```

Comment:
If it just fails to open a database, it returns an error here.
https://github.com/aquasecurity/trivy-db/blob/6100c2e3d9837c28b1f7f88d5e7c5358d4c258c1/pkg/db/db.go#L96-L98

As far as I know, it panics only when the database is broken.

```suggestion
			err = eb.Errorf("db corrupted: %s", r)
```

Reply:
I don't have as much experience as you, so I'll trust your information.